we don't support running dinit system-wide,
so remove the code that assumes system dinit

diff --git a/backend/dinit b/backend/dinit
index d69710e..7c5e680 100644
--- a/backend/dinit
+++ b/backend/dinit
@@ -85,7 +85,6 @@ set -e
 # set a bunch of defaults in case the conf cannot be read or is mangled
 
 [ -z "$boot_dir" ] && boot_dir="${HOME}/.config/dinit.d/boot.d"
-[ -z "$system_boot_dir" ] && system_boot_dir="/usr/lib/dinit.d/user/boot.d"
 
 if [ -z "$services_dir1" ]; then
     services_dir1="${HOME}/.config/dinit.d"
@@ -113,10 +112,4 @@ depends-on = system
 waits-for.d = ${boot_dir}
 EOF
 
-# this must also succeed
-cat << EOF > "${DINIT_DIR}/system"
-type = internal
-waits-for.d = ${system_boot_dir}
-EOF
-
 exec dinit --user --ready-fd 3 --services-dir "$DINIT_DIR" "$@" 3>"$DINIT_READY_PIPE"
diff --git a/backend/dinit.conf b/backend/dinit.conf
index c41f6bc..fc174a3 100644
--- a/backend/dinit.conf
+++ b/backend/dinit.conf
@@ -13,12 +13,6 @@
 #
 boot_dir="${HOME}/.config/dinit.d/boot.d"
 
-# This is just like boot_dir, but not controlled by the
-# user. Instead, the system installs links there, and
-# they are started for all users universally.
-#
-system_boot_dir="/usr/lib/dinit.d/user/boot.d"
-
 # A directory user service files are read from. Every
 # additional directory needs to have its number incremented.
 # The numbering matters (defines the order) and there must be
diff --git a/meson.build b/meson.build
index 770b982..733cd40 100644
--- a/meson.build
+++ b/meson.build
@@ -111,14 +111,6 @@ pam_mod = shared_module(
     gnu_symbol_visibility: 'hidden'
 )
 
-if have_dinit
-    install_data(
-        'data/dinit/turnstiled',
-        install_dir: join_paths(get_option('sysconfdir'), 'dinit.d'),
-        install_mode: 'rw-r--r--'
-    )
-endif
-
 install_data(
     'data/pam/turnstiled',
     install_dir: pamdir,
